.circleci/config.yml

```diff
@@ -339,9 +339,9 @@ workflows:
       - deploy-to-ecs:
           requires:
             - docker-push
-#          filters:
-#            branches:
-#              only: /^release-2\.0.\d+$/
+          filters:
+            branches:
+              only: /^release*/
```

Comment:
Were we going to do `/^release-2*/` instead? There is currently a `release-1.17.0` branch and could at some point be `release-1.17.1`. Should branches like that overwrite staging deployment?

cc @spencern

Reply:
OK I'll change it to /^release-2.*/ for now.